pyteal/ast/abi/reference_type.py

```diff
@@ -57,37 +46,16 @@ def __init__(self, spec: ReferenceTypeSpec) -> None:
     def type_spec(self) -> ReferenceTypeSpec:
         return cast(ReferenceTypeSpec, super().type_spec())
 
-    def get(self) -> Expr:
-        return self.stored_value.load()
+    def referenced_index(self) -> Expr:
+        """Get the reference index for this value.
 
-    def set(
```

Comment:
I've chosen to remove the `set` method for now because it's not used or needed currently. Reference types aren't supposed to be user-created, so if we want to change that we'll have to revisit.